packages/venia-ui/lib/components/CartPage/ProductListing/productListingFragments.js

```diff
@@ -23,6 +23,7 @@ export const ProductListingFragment = gql`
                 configurable_options {
                     id
                     option_label
+                    value_id
```

Comment:
Silly enough - we now have two "required" fields. `id` per validation although it's not actually guaranteed to be unique and `value_id` because we use it for caching though you'd find no code within `CartPage/` that uses it.